flow/layers/image_filter_layer.cc

```diff
@@ -11,6 +11,35 @@ ImageFilterLayer::ImageFilterLayer(sk_sp<SkImageFilter> filter)
       transformed_filter_(nullptr),
       render_count_(1) {}
 
+#ifdef FLUTTER_ENABLE_DIFF_CONTEXT
+
+void ImageFilterLayer::Diff(DiffContext* context, const Layer* old_layer) {
+  DiffContext::AutoSubtreeRestore subtree(context);
+  auto* prev = static_cast<const ImageFilterLayer*>(old_layer);
+  if (!context->IsSubtreeDirty()) {
+    FML_DCHECK(prev);
+    if (filter_ != prev->filter_) {
+      context->MarkSubtreeDirty(context->GetOldLayerPaintRegion(old_layer));
+    }
+  }
+
+  DiffChildren(context, prev);
+
+  SkMatrix inverse;
+  if (context->GetTransform().invert(&inverse)) {
+    auto paint_bounds = context->CurrentSubtreeRegion().GetBounds();
+    inverse.mapRect(&paint_bounds);
+    paint_bounds = filter_->computeFastBounds(paint_bounds);
```

Comment:
See comment above in backdropfilter about `computeFastBounds` vs `filterBounds`

Reply:
Yes. I mentioned this above, this seems wrong.